plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts

```diff
@@ -155,6 +161,7 @@ export default function transformProps(chartProps: ChartProps): EchartsProps {
         showMaxLabel: xAxisShowMaxLabel,
         formatter: xAxisFormatter,
       },
+      nameRotate: 50,
```

Comment:
Could we add this one to the constants, too?

Reply:
Removed